recipes-core/images/xenclient-stubdomain-initramfs-image.bb

```diff
@@ -14,7 +14,7 @@ IMAGE_INSTALL = " \
     busybox \
     bridge-utils \
     initramfs-stubdomain \
-    xen-xenstore \
+    xen-tools-xenstore \
```

Comment:
Wait, I though you were switching to virtual/xenstore? Shouldn't you include virtual/xenstore and then allow preferred provider to kick in?

Reply:
`xenstore` and `xenstored` are different packages, for the client and daemon respectively. Only `xenstored` is virtual.

Reply:
Is this related to above where we are building both C and Ocaml to mix and match xenstore and xenstored implementations?

Reply:
> Is this related to above where we are building both C and Ocaml to mix and match xenstore and xenstored implementations?

No, this specific change is just an update after the packages were renamed as part of the recipe separation.

However, on a similar topic, I've just pushed a few added commits with cleanups to the handling of the C xenstore daemon and the preference selection config which determines which is installed. I think I've addressed what's needed for this here.